config/owasp/suppressions.xml

```diff
@@ -8,4 +8,12 @@
     <packageUrl regex="true">^pkg:maven/org\.springframework/spring\-.*?@[1-5]\..*$</packageUrl>
     <cve>CVE-2016-1000027</cve>
   </suppress>
+  <suppress until = "2022-10-15">
+    <notes><![CDATA[
+        file name: jackson-databind-2.11.4.jar
+        ]]></notes>
+    <packageUrl regex="true">^pkg:maven/com\.fasterxml\.jackson\.core/jackson\-databind@.*$</packageUrl>
+    <cve>CVE-2022-42003</cve>
+    <cve>CVE-2022-42004</cve>
+  </suppress>
```

Comment:
Maybe put it above? Keeps the suppressions in ascending order by date

Reply:
done